crates/autopilot/src/infra/persistence/fee_policy/mod.rs

```diff
@@ -0,0 +1,41 @@
+use {
+    crate::{driver_model::solve::FeePolicy, protocol::fee},
+    anyhow::Context,
+    database::byte_array::ByteArray,
+    model::order::OrderUid,
+};
+
+impl super::Persistence {
+    pub async fn store_fee_policies(
```

Comment:
Interface should probably take the domain::auction and the order_uid iterator for matched orders only and then fetch the auction ID and fee policies from that.

Reply:
Everything was fixed after the merge